cluster/userdata-master.yaml

```diff
@@ -810,6 +813,67 @@ write_files:
               - kubectl
               - proxy
 
+  - path: /srv/kubernetes/manifests/services/kube-apiserver-readonly-svc.yaml
+    content: |
+      apiVersion: v1
+      kind: Service
+      metadata:
+        name: apiserver-svc-ro
+        namespace: kube-system
+      spec:
+        type: ClusterIP
+        ports:
+          - port: 80
+            protocol: TCP
+            targetPort: 8082
+        selector:
+          application: kube-apiserver
+          version: v1.4.5_coreos.0
+
+  - path: /srv/kubernetes/manifests/deployments/kube-state-metrics.yaml
+    content: |
+      apiVersion: extensions/v1beta1
+      kind: Deployment
+      metadata:
+        name: kube-state-metrics
+        namespace: kube-system
+      spec:
+        replicas: 1
+        template:
+          metadata:
+            labels:
+              application: kube-state-metrics
+              version: v0.3.3
+          spec:
+            containers:
+            - image: registry.opensource.zalan.do/teapot/kube-state-metrics:v0.3.3
+              name: kube-state-metrics
+              ports:
+              - containerPort: 8080
+              command:
+              - /kube-state-metrics
+              - --port=8080
+              - --apiserver=http://apiserver-svc-ro.kube-system.svc.cluster.local
+
+  - path: /srv/kubernetes/manifests/services/kube-state-metrics.yaml
+    content: |
+      apiVersion: v1
+      kind: Service
+      metadata:
+        name: kube-state-metrics
+        namespace: kube-system
+        annotations:
+          service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
+      spec:
+        type: LoadBalancer
+        ports:
+        - port: 80
+          protocol: TCP
+          targetPort: 8080
+        selector:
+          application: kube-state-metrics
+          version: v0.3.3
```

Comment:
the other services in this file usually don't put the `version` in the selector of the service